buildh/cli.py

```diff
@@ -117,21 +120,30 @@ def parse_cli():
     Handle the user parameters from the command line.
     """
 
+    # Retrieve list of supported lipids
+    lipids_files = [ f for f in PATH_JSON.iterdir() if f.is_file()]
```

Comment:
Haha, I like very much your use of comprehension list 😄 One minor syntax remark: to be consistent, I suggest either you do not put a space `[i for i in range(3)]` or you put one at the beginning and at the end `[ i for i in range(3) ]`.

Reply:
I think the i) is more PEP8 compatible. I will remove the paces at the beginning/end. Good catch!